.gitignore

```diff
@@ -0,0 +1,109 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
The `.gitignore` file is copied verbatim from https://github.com/espressif/github-actions. Now that `sync-pr-to-gitlab` is a standalone repo, are the entries in `.gitignore` still relevant? Or is the `.gitignore` file going to be standardized across all Github action repos?

Reply:
@Dazza0 You are right .... `.gitignore` stripped to necessary minimum relevant to the project. Thx!